docs/user/installation.rst

```diff
@@ -68,6 +69,17 @@ Optional dependencies:
    zypper in bash-completion jq
    zypper in python3-psycopg2 python3-pymongo python3-sleekxmpp
 
+Docker (beta)
+^^^^^^^^^^^^^
+
+**ATTENTION** Currently docker version can differ from intelmq version namings.
+**ATTENTION** If you use Docker you cant access :doc:`intelmqctl` via cli. You need to use `IntelMQ-Manager <https://github.com/certtools/intelmq-manager>`_ currently!
+
```

Comment:
s/access/run
s/via/from the
and I would specify: You can't manage you IntelMQ botnet (which is inside the container) using `intelmqctl` from outside the container.